src/SolARDescriptorsExtractorFromImagePopSift.cpp

```diff
@@ -153,15 +154,15 @@ FrameworkReturnCode SolARDescriptorsExtractorFromImagePopSift::extract(
     if (allocTestError!=PopSift::AllocTest::Ok)
         LOG_ERROR("{}",m_popSift->testTextureFitErrorString(allocTestError,image->getWidth(), image->getHeight()));
 
-    SiftJob* job;
+    std::unique_ptr<SiftJob> job;
     if (m_imageMode == "Unsigned Char")
-        job = m_popSift->enqueue(image->getWidth(), image->getHeight(), (unsigned char*)greyImage->data());
+        job.reset(m_popSift->enqueue(image->getWidth(), image->getHeight(), (unsigned char*)greyImage->data()));
     else if (m_imageMode == "Float")
-        job = m_popSift->enqueue(image->getWidth(), image->getHeight(), (float*)greyImage->data());
+        job.reset(m_popSift->enqueue(image->getWidth(), image->getHeight(), (float*)greyImage->data()));
     else
         return FrameworkReturnCode::_ERROR_;
 
-    popsift::FeaturesHost* popFeatures = job->getHost();
+    std::unique_ptr<popsift::Features> popFeatures(job->get());
```

Comment:
I'm not familiar with the API so this is a naive question: was that intented to move from `FeaturesHost` to `Features` ?

Reply:
No idea about this change. but the latest version of popsift code no longer uses FeaturesHost. https://github.com/alicevision/AliceVision/blob/develop/src/aliceVision/feature/sift/ImageDescriber_SIFT_popSIFT.cpp#L39
https://github.com/alicevision/popsift/issues/140